controllers/workspace/devworkspace_controller.go

```diff
@@ -123,7 +123,10 @@ func (r *DevWorkspaceReconciler) Reconcile(ctx context.Context, req ctrl.Request
 		return reconcile.Result{}, err
 	}
 
-	config := wkspConfig.GetGlobalConfig()
+	config, err := wkspConfig.ResolveConfigForWorkspace(rawWorkspace, clusterAPI.Client)
+	if err != nil {
+		reqLogger.Error(err, "Error applying external DevWorkspace-Operator configuration")
+	}
 	configString := wkspConfig.GetCurrentConfigString()
```

Comment:
`wkspConfig.GetCurrentConfigString()` should be updated to take a config as a parameter, to make it easier to print the resolved config. As it stands, using a custom config doesn't change the log below:
```go
reqLogger.Info("Reconciling Workspace", "resolvedConfig", configString)
```

Reply:
Fixed by 41929c25